src/psyclone/dynamo0p3_builtins.py

```diff
@@ -595,13 +595,29 @@ def __str__(self):
     def gen_code(self, parent):
         from psyclone.f2pygen import AssignGen
         # In this case we're raising each element of a field to a
-        # supplied scalar value.
+        # supplied real scalar value.
         field_name = self.array_ref(self._arguments.args[0].proxy_name)
         real_power = self._arguments.args[1].name
         parent.add(AssignGen(parent, lhs=field_name,
                              rhs=field_name + "**" + real_power))
 
 
+class DynIncXPowintNKern(DynBuiltIn):
+    ''' Raise a field to an integer power and return it '''
+
+    def __str__(self):
+        return "Built-in: raise a field to an integer power"
+
+    def gen_code(self, parent):
+        from psyclone.f2pygen import AssignGen
```

Comment:
Please document the interface to this routine.

Reply:
Documented interface to all "def gen_code" routines in the file.